applications/ContactMechanicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -38,48 +38,48 @@ namespace Python
 void  AddCustomProcessesToPython(pybind11::module& m)
 {
 
-  using namespace pybind11;
+  namespace py = pybind11;
```

Comment:
changes in the Contact mechanics do not belong here

Reply:
@RiccardoRossi look at the outdated, for some absurd reason GitHub shows updated changes